messages/jsonschema/Pickle.json

```diff
@@ -43,6 +44,16 @@
           "description": "A unique ID for the PickleStep",
           "type": "string"
         },
+        "type": {
+          "description": "The context in which the step was specified: context (Given), action (When) or outcome (Then).\n\nNote that the keywords `But` and `And` inherit their meaning from prior steps and the `*` 'keyword' doesn't have specific meaning (hence Unknown)",
+          "type": "string",
+          "enum": [
+            "Context",
+            "Action",
+            "Outcome",
+            "Unknown"
```

Comment:
See comment about unknown being first to make It 'default'

Reply:
Changed too.